plugins/ssr/ssrequality.ml

```diff
@@ -433,15 +433,15 @@ let lz_coq_prod =
 
 let lz_setoid_relation =
```

Comment:
What is the purpose of this function? It looks like a cache but the implementation is a bit fishy and there are no comments...